### ISSUE TITLE
mp.linalg.mineig?

### ISSUE THREAD

**Issue by dseuss** [dseuss/mpnum#11]
What is mineig actually computing? If it really is the minimal eigenvalue, the following example should give 0

    In [1]: import mpnum as mp
    In [2]: a = mp.mps_to_mpo(mp.random_mps(6, 4, 1))
    In [3]: mp.linalg.mineig(a, eigs_opts={'maxiter': 5000})
    Out[3]:
    ((0.99999999999999956-4.9034074076101329e-17j),
      <mpnum.mparray.MPArray at 0x1067ceba8>)

**Comment by dseuss** [comment]
Fixed by 347b572476a9b4e406cae8b510ecc0c1d945d90c

**Comment by milan-hl** [comment]
Thanks for noticing the issue and closing the bug. I was about to write a lengthy comment when I was interrupted:

It would have computed the minimal eigenvalue if you had added `which='SR'` to `eigs_opts`. I consider this a bug and have changed it accordingly, your example now gives:

```
In [1]: import mpnum as mp
In [2]: a = mp.mps_to_mpo(mp.random_mps(6, 4, 1))
In [3]: mp.linalg.mineig(a, eigs_opts={'maxiter': 5000}, startvec_bonddim=2)
Out[3]: 
((-4.1299208747024751e-17+1.6355024843943488e-17j),
 <mpnum.mparray.MPArray at 0x7f7b314ff400>)
```

I have changed the bond dimension of the eigenvector from 1 to 2 because mineig() currently does not support bond dimension 1 for eigenvectors.
